orca-sql/src/main/kotlin/com/netflix/spinnaker/orca/notifications/SqlNotificationClusterLock.kt

```diff
@@ -52,8 +53,19 @@ class SqlNotificationClusterLock(
       jooq.insertInto(lockTable)
         .set(lockField, notificationType)
         .set(expiryField, now.plusSeconds(lockTimeoutSeconds).toEpochMilli())
-        .onDuplicateKeyIgnore()
-        .execute()
+        .run {
+          when (jooq.dialect()) {
+            SQLDialect.POSTGRES -> {
+              onConflict(DSL.field("id"))
```

Comment:
This should be `lock_name`, not `id` which is not a field in this table. `lock_name` is the primary key.

Reply:
This is fixed, thank you.